src/robot/handler/protobuf.py

```diff
@@ -24,12 +27,16 @@ async def dispatch_message(self, bus_message: BusMessage) -> None:
         type_msg = bus_message.WhichOneof("message_content")
         if type_msg == "heartbeat":
             pass
```

Comment:
suggestion : if a heartbeat isn't received for a board for a few seconds, the board has an issue, log it.

Reply:
Should be in a separate commit, created an issue https://github.com/outech-robotic/hl-flowing-clean-arch/issues/88